DesktopModules/Vanjaro/UXManager/Extensions/Menu/Pages/Views/Setting/Detail.js

```diff
@@ -383,7 +383,7 @@ app.controller('setting_detail', function ($scope, $routeParams, CommonSvc, Swee
                         var ParentScope = parent.document.getElementById("iframe").contentWindow.angular;
                         if (ParentScope != null && ParentScope != undefined) {
                             var Menuextension = ParentScope.element(".menuextension");
-                            if (ParentScope != undefined && Menuextension != undefined && Menuextension.scope() != undefined && Menuextension.scope().FetchPages != undefined) {
+                            if (Menuextension != undefined && Menuextension.scope() != undefined && Menuextension.scope().$apply != undefined && Menuextension.scope().init != undefined && Menuextension.scope().FetchPages != undefined && Menuextension.scope().Findnode != undefined && Menuextension.scope().ui != undefined && Menuextension.scope().ui.data != undefined && Menuextension.scope().ui.data.PagesTree != undefined) {
```

Comment:
This doesn't make sense. There has to be a better way to check for undefined; perhaps use the Menuextension.scope().ui.data.PagesTree in a try catch or utility function? See: https://stackoverflow.com/questions/23808928/javascript-elegant-way-to-check-nested-object-properties-for-null-undefined

@Gaganbird Please assist.